spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/JSqlParserQueryEnhancer.java

```diff
@@ -120,7 +121,7 @@ public String applySorting(Sort sort, @Nullable String alias) {
 
 		Select selectStatement = parseSelectStatement(queryString);
 
-		if (selectStatement.getSelectBody()instanceof SetOperationList setOperationList) {
+		if (selectStatement.getSelectBody() instanceof SetOperationList setOperationList) {
 			return applySortingToSetOperationList(setOperationList, sort);
```

Comment:
Whereas I sympathize with the desire to introduce this space WHICH SHOULD BE HERE, our current tools, due to `getSelectBody()` (the parentheses) will take that space back out the next time I'm inside this class and I reformat it. Hence, I'd rather not introduce that churn.